ingest/ingest_pipeline.py

```diff
@@ -299,30 +271,39 @@ def ingest_cell_metadata(self):
             if self.kwargs['validate_convention'] is not None:
                 if self.kwargs['validate_convention']:
                     if self.has_valid_metadata_convention():
-                        print("it works!")
                         pass
                     else:
                         return 1
             self.cell_metadata.reset_file(2, open_as="dataframe")
             self.cell_metadata.preproccess()
             for metadataModel in self.cell_metadata.transform():
```

Comment:
```suggestion
            for metadata_model in self.cell_metadata.transform():
```

And similarly for other uses of this variable.